packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -547,6 +554,8 @@ class Gutter extends StatelessWidget {
     required this.executableLines,
     required this.onPressed,
     required this.allowInteraction,
+    required this.hitLines,
+    this.showCodeCoverage = true,
```

Comment:
should this default to false like the value notifier in the controller?

Reply:
This should just be required, I had it this way for testing during development.